README.md

```diff
@@ -35,27 +35,12 @@ enough data to span the desired prediction period. The checkpoint is also availa
 `gs://ai2cm-public-requester-pays/2023-11-29-ai2-climate-emulator-v1/checkpoints/ace_ckpt.tar`.
 
 ### 3. Update configuration and run
-Update the paths in the [example config](examples/config-inference.yaml). Then in the
+Update the paths in the [example config](fme/docs/inference-config.yaml). Then in the
 `fme` conda environment, run inference with:
 ```
-python -m fme.fcn_training.inference.inference examples/config-inference.yaml
+python -m fme.ace.inference fme/docs/inference-config.yaml
```

Comment:
I wanted it to show the inference entrypoint, since this has the least requirements and I expect it is how most people will use the model. For now, I think we should keep the quickstart, but I agree once the RTD is hosted we could just have a link instead.